### ISSUE TITLE
validation loss not decreasing after 25k steps, Fine-tuning with small target data 

### ISSUE THREAD

**Issue by mraj96** [bshall/acoustic-model#11]
@bshall Thank you for this great work.

validation loss not decreasing after 25k steps, Fine-tuning with small target data (~1-hour dataset).

the best model (25k steps) not working properly. content transferred ok, but the pitch was not converted from the source to the target. and the target speaker's tone does not close.

@bshall can you please give your suggestions?

Thanks

![image](https://user-images.githubusercontent.com/35978784/202907176-fde1c21a-5060-4776-830b-0fe82c376738.png)

**Comment by aeciorc** [comment]
What hyperparams did you use @mraj96?

**Comment by yukiarimo** [comment]
Any updates?
